airbyte-integrations/third-party/gjbakerross/airbyte-source-criteo/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG AIRBYTE_TO_FLOW_TAG="dev"
+FROM ghcr.io/estuary/airbyte-to-flow:$AIRBYTE_TO_FLOW_TAG as airbyte-to-flow
+
+FROM ghcr.io/estuary/gjbakerross/source-criteo:dev
+
+COPY *.json ./
+# COPY streams/* ./streams/
```

Comment:
I would recommend keeping this uncommented. It won't have any negative effect, but will prevent mistakes where we add stream patches without effect, and we have to figure out why the stream patches are not working

Reply:
Yeah so I had originally tried that and it didn't work, but I realized that I was building from the root of the project, not from the connector directory. I uncommented these lines!